packages/did/src/Did.rpc.ts

```diff
@@ -160,3 +162,121 @@ export function linkedInfoFromChain(
     didName,
   }
 }
+
+interface PalletDidDidDetailsApi extends Struct {
```

Comment:
Why are we implementing this again? DID resolver package is already doing this

Reply:
For the function 'linkedInfoFromApi'

Reply:
Yes but its actually not needed